dev-libs/boost/boost170-1.70.0.recipe

```diff
@@ -196,6 +200,11 @@ INSTALL()
 		--without-python \
 		install
 
+	prepareInstalledDevelLibs `echo "$static_libs" | sed -n \
+		-e "s/devel:\(.*\)$secondaryArchSuffix =.*/\1/p"`
+
+	rm $libDir/*.a
```

Comment:
Couldn't you just remove `,static` from the `link=` option to b2 above to not build the static libraries at all instead of deleting them here?

Reply:
Slipped my eye, will do a new build for that tomorrow :+1:

Reply:
Done, for boost169 3 static libraries remain that still are being installed, those are removed in the recipe.